lib/index.js

```diff
@@ -65,6 +66,11 @@ module.exports = class EyeglassCompiler extends BroccoliSassCompiler {
     this.assetDirectories = assetDirectories;
     this.assetsHttpPrefix = assetsHttpPrefix;
     this.events.on("compiling", this.handleNewFile.bind(this));
+    this.assetImportCache = {};
```

Comment:
nit-pick: In general, for caches `Object.create(null)` is more appropriate.

two reason:

* `Object.create(null)` puts the object straight into a dictionary mode, which its more conducive to a slowly growing or mutating objects.
* isn't as risk of pollution due to inherited properties that are added to `Object.prototype`

Reply:
ok, done